README.md

```diff
@@ -138,33 +137,52 @@ npm run setup
 
 The setup utility consist of a number of actions that should be performed.
 
-### Cloudant
+#### Cloudant
 
 This action will create the databases required to run the application as well as load all the design documents and indexes.  You need to have the Cloudant username and password handy, which you can find in the credentials of your Cloudant service in Bluemix.
 
-### Twitter
+#### Twitter
 
 This action will update the configuration with the parameters needed to connect to Twitter as well as subscribe to tweets.  You need to have the Consumer Key (API Key), Consumer Secret (API Secret), Access Token and Access Token Secret from dev.twitter.com available.
 
-At this point you should have decided what Twitter handle you will be "listening" to.  The Twitter handle would be something other people would tweet to, for example `@united`, `@AmericanAir`, `@JetBlue` (Please note that these are CASE SENSITIVE) etc.  This value is what Twitter would use as trigger to send you the tweets.
-
-When you selected the `Twitter` option on the main menu of the setup utility, you will be required to enter the Twitter Tokens first.  Then the screen name you are listening to, the Conversation API workspace id for the classifications and finally the Chatbot screen name and Conversation API workspace for the Dialog implementation.
+At this point you should have decided what Twitter handle you will be "listening" to.  The intents for this journey are configured to work with a Twitter account for an airline, but you could create relevant intents for whichever business domain you wish. The Twitter handle would be something other people would tweet to, for example `@aircanada`.  This value is what Twitter would use as trigger to send you the tweets.
```

Comment:
Perhaps clarify the use of "intents" here (that it's the service, not "the intention of this journey")

Reply:
Maybe link [@aircanada](https://twitter.com/aircanada) ?

Reply:
done